omod/src/main/webapp/fragments/patientdashboard/visits.gsp

```diff
@@ -103,7 +103,8 @@
                 startDateLowerLimit: (idx + 1 == visits.size || visits[idx + 1].stopDatetime == null) ? null : editDateFormat.format(org.apache.commons.lang.time.DateUtils.addDays(visits[idx + 1].stopDatetime, 1)),
                 startDateUpperLimit: wrapper.oldestEncounter == null && wrapper.stopDatetime == null ? editDateFormat.format(new Date()) : editDateFormat.format(wrapper.oldestEncounter == null ? wrapper.stopDatetime : wrapper.oldestEncounter.encounterDatetime),
                 defaultStartDate: wrapper.startDatetime,
-                defaultEndDate: wrapper.stopDatetime
+                defaultEndDate:idx == 0 ? null : wrapper.stopDatetime,
```

Comment:
What is the motivation for this change?

Reply:
This is to allow user to make a most recent Visit  endDate null so as to make it an active visit otherwise if the visit is behind another visit in time then you are forced to have a default end Date